fido_app/models.py

```diff
@@ -35,3 +38,37 @@ def set_password(self, password):
     def check_password(self, password):
         '''Checks if the given plaintext password matches the salt and hash for the given user'''
         return check_password_hash(self.password_hash, password)
+
+
+class Session(db.Model):
+    '''
+    Model associating users with persistent cookie tokens;
+    'user' property is implicitly created by relationship in User class
+    '''
+
+    id = db.Column(db.Integer, primary_key=True)
+    
+    user_id = db.Column(db.Integer, db.ForeignKey('user.id'))
+    token = db.Column(db.String(32), unique=False, nullable=False)
```

Comment:
We probably want session tokens to be unique, right?

Reply:
Yes; I left it as non-unique because I wasn't sure what we were going to use as tokens and how uniqueness was going to be enforced, and unique=False seemed less restrictive. It's changed now.